packages/victory-candlestick/src/victory-candlestick.js

```diff
@@ -113,6 +113,10 @@ class VictoryCandlestick extends React.Component {
     return !!this.props.animate;
   }
 
+  shouldRenderDatum(datum) {
+    return datum._x !== null && datum._x !== undefined;
+  }
```

Comment:
`VictoryBoxPlot` will need a similar `shouldRenderDatum` method

Reply:
VictoryBoxPlot doesn't use `addEvents.renderData()` so it shouldn't be affected by these changes. I took a quick stab at updating its internal method for creating data components and it's a little more hairy than these changes. Would it be better to update it in another PR?

Reply:
Ah, I see what you mean. I think it can stay with this PR. 

I don't think any of the parts of the boxplot should render if any of the values are null, so that should hopefully simplify the logic.